### ISSUE TITLE
question: apollo doshboard is really ugly, please ask if there is a plan to optimize the dashboard?

### ISSUE THREAD

**Issue by NICEXAI** [apolloconfig/apollo#4023]

**Comment by DiegoKrupitza** [comment]
What changes do you think will make the dashboard "beautiful" or optimize the experience?

**Comment by NICEXAI** [comment]
Such as: neat layout, no distortion in typography, unified style， etc.

**Comment by NICEXAI** [comment]
The right spacing may make the page look fuller.

**Comment by NICEXAI** [comment]
![image](https://user-images.githubusercontent.com/19580983/136655857-c8155d27-004f-456b-835c-725c8d7f2ea2.png)
No padding

![image](https://user-images.githubusercontent.com/19580983/136655923-36e24bec-33e9-4969-a8f9-b9807e07f650.png)

Oversized padding

**Comment by nobodyiam** [comment]
I agree there are rooms for the UI to improve, so any suggestion or pr is really welcome :-)
BTW, there is a pr #3850 which optimized the homepage a little.

**Comment by DiegoKrupitza** [comment]
I think a good way to start here is to create a prototype in Adobe XD/Figma, so we can collect feedback. This will also make a good `good-first-issue`.

**Comment by NICEXAI** [comment]
> I think a good way to start here is to create a prototype in Adobe XD/Figma, so we can collect feedback. This will also make a good `good-first-issue`.

I think it is not very realistic to redesign and implement the whole dashboard in a short time, but we can optimize it gradually.
